x-pack/plugins/security_solution/server/lib/telemetry/filterlists/prebuilt_rules_alerts.ts

```diff
@@ -0,0 +1,307 @@
+/*
```

Comment:
This is great, and hopefully the right set for now. We'll run it past TRADE team to make sure it's useful and double/triple check that it doesn't spill PII.